lib/core/presentation/pages/event/guest_event_detail_page/widgets/more_less_description.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/color.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class MoreLessDescription extends StatefulWidget {
+  const MoreLessDescription({
+    super.key,
```

Comment:
Consider using ExpandableNotifier for better animation when the text expanding or collapsing

Reply:
👍